app.css

```diff
@@ -6,12 +6,14 @@
 @import url("css/card.css");
 @import url("css/image.css");
 @import url("css/input.css");
+@import url("css/text.css");
+@import url("css/list.css");
 
 
 :root{
     --primary-color: rgb(81, 97, 116);
     --secondary-color:white;
-    --cover-color:rgb(224, 222, 222);
+    --cover-color:#c7dde6;
```

Comment:
use only one format for colors

Reply:
Yaah at last ,I'm gonna correct all the color issues collectively.